code/game/objects/items/devices/flashlight.dm

```diff
@@ -17,14 +17,33 @@
 	light_range = 4
 	light_power = 1
 	light_on = FALSE
+	/// Can we toggle this light on and off
+	var/can_toggle = TRUE
```

Comment:
It makes it sound like it's actually used to make sure something can be turned on/off, but all it does is decide whether the context should be applied to it. Please rename it or make it early return when trying to turn the flashlight off

Reply:
Renamed with 4dc5266aa103a223dc6a9d46b3cc99b1114f5841